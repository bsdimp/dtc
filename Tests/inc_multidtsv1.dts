/dts-v1/;
/include/ "inc1_multidtsv1.dtsi"


/ {
  foo {
    compatible = "simple-bus";
  };
};

