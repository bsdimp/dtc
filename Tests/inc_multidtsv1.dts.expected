/dts-v1/;

/  {

	foo {

		compatible = "simple-bus";
	};
};
